src/main/java/uk/gov/digital/ho/pttg/dto/IndividualForNameMatching.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.digital.ho.pttg.dto;
+
+import lombok.Data;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+@Data
+public class IndividualForNameMatching implements Serializable {
```

Comment:
I think it is correct to split this out as we probably should never have been reusing the Individual class from the HMRC domain in our API.  However you should probably move Individual out of this package and into our application package, and change this class to a better name - maybe HmrcIndividual?

Reply:
I have done this - also I have relocated Income and IncomeSummary to a new application.domain class to reflect that they are not DTOs